draft-ietf-ppm-dap.md

```diff
@@ -2654,6 +2654,11 @@ the input shares and defeat privacy.
 
 # IANA Considerations
 
+## Message versioning
+Media types for the HTTP requests are specific to this version of DAP. When a new major enhancement is proposed that results in newer IETF specification for DAP, a new set of media types need to be defined. In other words, newer versions of DAP will not be backward compatible with this version of DAP.
+
+HTTP requests with DAP media types MAY express an optional parameter 'version'. Value of this parameter indicates current draft version of the protocol the component is using. This MAY be used as a hint by the receiver of the request to do compatibility checks between client and server.
```

Comment:
I think this paragraph would not fit into the eventual RFC for DAP, since there will be no draft version to refer to. IMO, add a TODO to remove this before RFC.

Reply:
Agreed. Concretely, add something like this: 
```suggestion
(NOTE TO RFC EDITOR: Remove this paragraph.) HTTP requests with DAP media types MAY express an optional parameter 'version'. Value of this parameter indicates current draft version of the protocol the component is using. This MAY be used as a hint by the receiver of the request to do compatibility checks between client and server.
```

Reply:
I'd like to see an example of this optional `version` parameter. Would this be something like `Media-Type: application/dap-aggregation-job-init-req;version=11`? A reference to [RFC 9110 section 8.3](https://www.rfc-editor.org/rfc/rfc9110.html#section-8.3.1) (which describes this kind of thing in media types) also would help.

Reply:
incorporated both suggestions.